clang/test/SemaCXX/constant-expression-cxx14.cpp

```diff
@@ -1306,3 +1306,18 @@ constexpr int field(int a) {
 static_assert(field(3), ""); // expected-error {{constant expression}} \
                              // expected-note {{in call to 'field(3)'}}
 }
+
+namespace literal_comparison {
+
+constexpr bool different_in_loop(bool b = false) {
+  if (b) return false;
```

Comment:
What's the purpose of this 'b' parameter?

Reply:
Just to suppress the "constexpr function never produces a constant expression" diagnostic.